contracts/abstract/JBProjectPayer.sol

```diff
@@ -10,82 +10,90 @@ import './../libraries/JBTokens.sol';
 // --------------------------- custom errors -------------------------- //
 //*********************************************************************//
 error INSUFFICIENT_BALANCE();
-error PROJECT_NOT_FOUND();
 error TERMINAL_NOT_FOUND();
 
 /** 
-  @notice A contract that inherits from JuiceboxProject can use Juicebox as a business-model-as-a-service.
-  @dev The owner of the contract makes admin decisions such as:
-    - Which address is the funding cycle owner, which can withdraw funds from the funding cycle.
-    - Should this project's Tickets be migrated to a new TerminalV1. 
+  @notice A contract that sends funds to a Juicebox project.
 */
-abstract contract JBProject is Ownable {
+abstract contract JBProjectPayer is Ownable {
   /// @notice The direct deposit terminals.
   IJBDirectory public immutable directory;
 
   /// @notice The ID of the project that should be used to forward this contract's received payments.
-  uint256 public projectId;
+  uint256 public defaultProjectId;
 
   /** 
-    @param _projectId The ID of the project that should be used to forward this contract's received payments.
-    @param _directory A directory of a project's current Juicebox terminal to receive payments in.
+    @param _defaultProjectId The ID of the project that should be used to forward this contract's received payments.
+    @param _directory The direct deposit terminals.
   */
-  constructor(uint256 _projectId, IJBDirectory _directory) {
-    projectId = _projectId;
+  constructor(uint256 _defaultProjectId, IJBDirectory _directory) {
     directory = _directory;
+    defaultProjectId = _defaultProjectId;
   }
 
   /** 
     Received funds go straight to the project.
   */
   receive() external payable {
-    _pay(msg.sender, '', false, JBTokens.ETH);
+    if (defaultProjectId > 0)
```

Comment:
shouldn't we else revert() ? Worried of stuck eth

Reply:
yes. great.